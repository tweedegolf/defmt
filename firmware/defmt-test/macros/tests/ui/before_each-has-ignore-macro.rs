fn main() {}

#[defmt_test_macros::tests]
mod tests {
    #[before_each]
    #[ignore]
    fn init() {}
}
