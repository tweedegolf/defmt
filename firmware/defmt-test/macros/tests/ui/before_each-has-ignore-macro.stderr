error: `#[ignore]` is not allowed on the `#[before_each]` function
 --> tests/ui/before_each-has-ignore-macro.rs:7:8
  |
7 |     fn init() {}
  |        ^^^^
